themes/build/layouts/_default/list.html

```diff
@@ -10,6 +10,11 @@
               {{- with (index . 0) }}{{ $tagname = index . "name"}}{{ end -}}
           {{- end -}}
           <h1>Articles by tag: {{ $tagname }}</h1>
+          {{ if (eq .Page.Params.title "core-weekly") }}
+            <div class="alert alert-info">
+              Starting December 2022 the Core Weekly is replaced by <a href="{{ relref . "/tags/core-monthly" }}">Core Monthly</a>
```

Comment:
Wouldn't it be easier just to retag previous core weeklies with core-monthly?

Reply:
We could do that and then redirect `core-weekly` do `core-monthly` to avoid having two pages with almost the same content.